src/main/java/Duke.java

```diff
@@ -1,10 +1,174 @@
+import java.util.Scanner;
+
 public class Duke {
+    private static Task[] allTasks;
```

Comment:
I like how you used plural form when naming this variable as it gives the reader the implicit understanding that this variable type is some collection of objects.